workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -287,24 +288,74 @@ describe('DynamicAuthorizationService', () => {
   });
 
   describe('addUserToGroup', () => {
-    it('returns userID and groupID when user was successfully added to the group', async () => {
-      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue({ data: { userId, groupId } });
+    let auditServiceWriteSpy: jest.SpyInstance;
+    let actor: object;
+    let source: object;
+    let action: string;
```

Comment:
Auditing will be done in other functions as well, can we move this to the outermost `describe` block?